index.css

```diff
@@ -82,3 +81,149 @@ body {
 .social-media-spacer {
   margin-left: 20px;
 }
+
+.grid-item {
+  display: block;
+  box-shadow: 2px;
+  padding: 5px;
+  width: 300px;
+  border: 2px solid#DFE1E6;
+  border-radius: 10px;
+}
+
+.image-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.image-wrapper > img {
+  width: 250px;
+  border-radius: 30px;
+  align-content: center;
+}
+
+.grid-container {
+  margin-top: 150px;
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-left: 15px;
+  row-gap: 70px;
+}
+
```

Comment:
Please use the right margins and paddings ie, spacing required for your project, the `my project` section of your project is really close to the social icons in the previous sections, the `a daily selection` paragraph color you used is not the right color, the `technologies used` section has a wider border than the one you used and the `See project` has a bigger `padding` size than the one you used. Kindly ensure you replicate your design as closely as possible to the given design to attract visitors and possible recruiters to a nice portfolio 😄 

Your design :

![image](https://user-images.githubusercontent.com/20537712/235315724-795de3fb-901e-4d10-8958-3531ba399dfc.png)

figma design 👍 

![image](https://user-images.githubusercontent.com/20537712/235315826-b546855c-b111-4d52-8f28-1e287b6bd9f4.png)